dashboard/app/models/levels/level.rb

```diff
@@ -169,12 +169,6 @@ def finishable?
     !unplugged?
   end
 
-  # This does not include DSL levels which also use teacher markdown
-  # but access it in a different way
-  def include_teacher_only_markdown_editor?
```

Comment:
This was added in this commit, but is no longer used. The `teacher_only_markdown` partial is included by each individual level type's HAML view, rather than in the shared level edit HAML view like it was when this commit was made.

https://github.com/code-dot-org/code-dot-org/commit/8d4905086d91847614357c4a7035937ef6ddb836